core/src/modules/ops/runs.ts

```diff
@@ -196,37 +195,22 @@ export namespace RunOps {
       });
 
       if (totalGroupMembers === 0 && membersAlreadyUpdated === 0) return 0;
+      if (group.state === "deleted") return 99;
```

Comment:
What's the thought process behind this?

Reply:
This was the only logical place in the code that used data in the Log table - when deleting a group, we used the previous size from the log to know what % we were done.  Now we can't do that, but I didn't think it was a valuable feature.  So... we are always 99% done :D

Reply:
I think to get this working again we would need to add a column to the groups table to store `lastRunGroupMemberCount` or something like that